romancal/tweakreg/astrometric_utils.py

```diff
@@ -180,18 +170,13 @@ def get_catalog(ra, dec, sr=0.1, catalog=DEF_CAT):
         CSV object of returned sources with all columns as provided by catalog
 
     """
-    service_type = "vo/CatalogSearch.aspx"
-    spec_str = "RA={}&DEC={}&SR={}&FORMAT={}&CAT={}&MINDET=5"
-    headers = {"Content-Type": "text/csv"}
-    fmt = "CSV"
-
-    spec = spec_str.format(ra, dec, sr, fmt, catalog)
-    service_url = f"{SERVICELOCATION}/{service_type}?{spec}"
-    rawcat = requests.get(service_url, headers=headers)
-    r_contents = rawcat.content.decode()  # convert from bytes to a String
-    rstr = r_contents.split("\r\n")
-    # remove initial line describing the number of sources returned
-    # CRITICAL to proper interpretation of CSV data
-    del rstr[0]
-
-    return Table.read(rstr, format="csv")
+    try:
+        catalog_data_csv = Catalogs.query_object(
+            f"{ra} {dec}", radius=sr, catalog=catalog, format="csv"
+        )
+        catalog_data_csv.rename_column("source_id", "objID")
+        catalog_data_csv.rename_column("phot_g_mean_mag", "mag")
+    except Exception as e:
+        raise ValueError("An error ocurred...").with_traceback(e.__traceback__)
```

Comment:
`"An error ocurred..."` as an error message seems very uninformative. Could this be improved? Maybe "Error retrieving data from VO web service"? Something more useful?

Reply:
The traceback with the specific error will be printed after that `An error occurred...` message.
Anything that raises an exception in that try block (not only from data retrieving) will have the appropriate traceback and error printed out.

I will refactor it to make the error message clearer by passing whatever exception is raised in the `try` block directly to the traceback.